### ISSUE TITLE
Save to file even if the job fails

### ISSUE THREAD

**Issue by fam4r** [pa11y/pa11y-ci#97]
Use case: in a CI Job it would be nice to have an artifact with reported errors/warnings/etc. and also let the job fail.

At the moment, using the redirection (eg. `> accessibility-report.html`), pa11y-ci fails and it doesn't write anything to file.

Using `pa11y-ci` v2.3.0 and `pa11y-reporter-html` in a GitLab pipeline,

**Comment by aarongoldenthal** [comment]
@fam4r If it help, [this](https://gitlab.com/gitlab-ci-utils/gitlab-pa11y-ci) is a project I maintain for running `pa11y-ci` in GitLab and the README has a handful of example GitLab CI jobs.
